nathandickinson32/second-nature
# feat: adding dependencies and first unit test �

To test:
git pull origin main
git checkout unitTestsPt2
npm install
(hopefully) npm run test

Should see 3 successful unit tests complete! 🥳
